src/stores/ExploreDataStore.ts

```diff
@@ -0,0 +1,130 @@
+import { makeAutoObservable } from "mobx";
+import { ConstraintInstance, TypeFormatTuple, WorkflowConfig, WorkflowSolution } from "./WorkflowTypes";
+import { makePersistable } from "mobx-persist-store";
+
+const emptyWorkflowConfig = () => {
```

Comment:
We should probably have at some point a default config (file) per domain, which includes the default inputs and outputs and is used to setup this const.

Reply:
That makes sense indeed. We could do this when we refactor / fix the handling of domain configs